City-of-Pori/pori-kada
# PORI-307 Added new theme color blueLight

To test:

1. drush cc all
2. Go to https://local.visitpori.fi/admin/structure/menu/manage/menu-visitpori-main-menu and edit the menu item "Järjestä tapahtuma". Edit the menu items attributes and add the following class: visitpori-theme-color-blueLight
3. Save and go to see on the front page that the menu item has changed color to light blue.
